app/test/testutils.cpp

```diff
@@ -128,6 +139,81 @@ QString TestUtils::generatePassword()
   return QStringLiteral( "_Pass12%1" ).arg( pass );
 }
 
+QString TestUtils::generateWorkspaceName( const QString &username )
+{
+  /*
+  * Create a workspace name from the generated username
+  * Output: a workspace name: mmat-DayMonthYear-HourMinutes
+  */
+  static const QRegularExpression regex( R"(merginautotest(\d{6})-(\d{4})\d{2}-\d{3})" );
```

Comment:
Let's not use the outdated `mergin` name, how about

```suggestion
  static const QRegularExpression regex( R"(mobile-autotest(\d{6})-(\d{4})\d{2}-\d{3})" );
```

Reply:
Aaand let's also update the format in `generateUsername` so that it does not use `Input` anymore